packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -205,7 +206,7 @@ const CollapsedActions = decorate([
       <DropdownMenu className='dropdown-menu-right'>
         {state.actions.map((action, key) => (
           <MenuItem disabled={action.disabled} key={key} onClick={() => effects.execute(action)}>
-            <Icon icon={action.icon} /> {action.label}
+            <Icon className={styles[action.level]} icon={action.icon} /> {action.label}
```

Comment:
Would be better if the color of the writing was the same as the color of the icon?

Reply:
I find that with the colored text, the text becomes less visible. I add a screenshot of the result in the `Screenshot section`